src/components/Footer/index.tsx

```diff
@@ -165,33 +225,29 @@ const Footer = () => {
             <CustomAccordion items={accordionItems} />
           </div>
           <div className={styles.Box} data-testid='get-connected'>
-            <Text type='subtitle-1' as='h3' className={styles.SectionTitle}>
+            <Heading.H5 className={styles.SectionTitle2}>
               <Translate>Get connected</Translate>
-            </Text>
+            </Heading.H5>
             <p className={styles.SectionContent}>
-              <Translate>Discuss ideas and share solutions with developers worldwide.</Translate>
+              <Text size='sm'>
+                <Translate>Discuss ideas and share solutions with developers worldwide.</Translate>
+              </Text>
             </p>
             <div className={styles.CommunityButton} data-testid='community-button'>
               <Button
-                color='secondary'
-                size='md'
-                rounded='md'
-                type='button'
-                variant='outlined'
-                className={styles.PaddedButton}
+                variant='secondary'
+                color='black'
+                fullWidth
                 onClick={() => {
                   window.open('https://deriv.vanillacommunity.com/', '_blank');
                 }}
               >
                 <Translate>Join our community</Translate>
               </Button>
               <Button
-                color='secondary'
-                size='md'
-                rounded='md'
-                type='button'
-                variant='outlined'
-                className={styles.PaddedButton}
+                variant='secondary'
+                color='black'
+                fullWidth
                 onClick={() => {
                   window.open('https://t.me/derivdotcomofficial', '_blank');
```

Comment:
@agrim-deriv please update the link as mentioned.
https://app.clickup.com/t/20696747/DAPI-720